cfme/control/explorer/alerts.py

```diff
@@ -304,6 +306,29 @@ def create(self, description, severity=None, active=None, based_on=None, evaluat
         view.flash.assert_success_message('Alert "{}" was added'.format(alert.description))
         return alert
 
+    def all(self):
+        # get alerts via reading 'All Alerts table'
+        view  = navigate_to(self.appliance.collections.alerts,'All')
+        try:
+            alert_table = view.table
```

Comment:
**Optional** You don't need to set this variable since you are only using it once below. Just update the for statement below:
```for alert in view.table.read()```